### PR TITLE
Close CFP and remove links

### PR DESCRIPTION
**Pull request by plaindocs**
----
📚 Documentation preview 📚: https://writethedocs-www--2077.org.readthedocs.build/

### CODE REVIEW

File: docs/_data/portland-2024-config.yaml [hunk 1]
```diff
@@ -43,12 +43,12 @@ buttons:
     # - text: Conference recap
     #   link: /news/thanks-recap
   bottom:
-    # - text: Welcome to our conference
-    #   link: /news/welcome
+    - text: Welcome to our conference
+      link: /news/welcome
```
Comment (ericholscher) [inline docs/_data/portland-2024-config.yaml]: I think we probably just want a ticket link here too, and not a link to our initial news post.

Comment (ericholscher) [reply docs/_data/portland-2024-config.yaml]: Pushed up a fix 👍

**Review by ericholscher — APPROVED** [review]

File: docs/_data/portland-2024-config.yaml (hunk 1 above)
Comment (plaindocs) [reply docs/_data/portland-2024-config.yaml]: Sounds good - merge when you're fully futzed?

File: docs/_data/portland-2024-config.yaml [hunk 2]
```diff
@@ -155,11 +155,11 @@ about:
...
-  url: https://pretalx.com/wtd-portland-2024/cfp  # Open November 6 (Mon) -> 9 weeks to submit
-  ends: "January 15, 2024"                        # (Mon) 1 week, to review. It'll take a little longer to announce
-  notification: "January 30, 2024"                # (Mon) 8 weeks to write and prepare
-  slides_by: "April 7, 2024"                      # (Mon) 1 week before
-  preview: "TODO"                                 # Preview URL TODO
+  url: https://pretalx.com/wtd-portland-2024/cfp # Open November 6 (Mon) -> 9 weeks to submit
+  ends: "January 15, 2024" # (Mon) 1 week, to review. It'll take a little longer to announce
+  notification: "January 30, 2024" # (Mon) 8 weeks to write and prepare
+  slides_by: "April 7, 2024" # (Mon) 1 week before
+  preview: "TODO" # Preview URL TODO
```
Comment (plaindocs) [inline docs/_data/portland-2024-config.yaml]: I feel like this is not an improvement ;-)

Comment (ericholscher) [reply docs/_data/portland-2024-config.yaml]: Ah yea.. didn't catch that it removed the spacing that was intentional... Really wish it standardized the quotes and whatnot, would be nice to have a good YAML linter.

Comment (plaindocs) [reply docs/_data/portland-2024-config.yaml]: Aye.

I guess we could also look at 

```
comment
comment 
comment

info 
info 
info
```

**Pull request merged**